src/pages/Lend/Filter/LendFilterMenu.vue

```diff
@@ -78,11 +80,13 @@
 
 <script>
 
+import algoliaConfig from '@/plugins/algolia-config-mixin';
```

Comment:
@foochris this is already included in the page. please pass down from there.

Reply:
@mcstover PR updated.